docs/components/Intro.js

```diff
@@ -447,10 +389,15 @@ function Explained() {
           </a>
           <div className="badge-purple">New</div>
         </div>
-        {displaySupportedTech()}
+        {SdkLogos()}
         <p className="landing-card-p">How-to guides for Temporal Application developers. </p>
         <ul className="landing-card-list">{displayLinks(appDevGuideLinks.listOne)}</ul>
         <ul className="landing-card-list">{displayLinks(appDevGuideLinks.listTwo)}</ul>
+        <p className="landing-card-p mt-4 mb-0">
+          For our .NET SDK, see the <a href="https://github.com/temporalio/sdk-dotnet#readme">README</a> and{" "}
+          <a href="https://dotnet.temporal.io/">API reference</a>. For our Ruby SDK, see the{" "}
```

Comment:
I'm inclined to push the second (Ruby) sentence onto a new line for scanability.